src/modules/endpoint/sagas/watchFetchEvent.js

```diff
@@ -5,31 +5,38 @@ import { all, call, put, takeLatest } from 'redux-saga/effects'
 import type { FetchEventActionType, FetchEventFailedActionType, PushEventActionType } from '../../app/StoreActionType'
 import type { DataContainer } from '../DataContainer'
 import loadCityContent from './loadCityContent'
+import { ContentLoadCriterion } from '../ContentLoadCriterion'
+import DatabaseContext from '../DatabaseContext'
 
 function * fetchEvent (dataContainer: DataContainer, action: FetchEventActionType): Saga<void> {
-  const {city, language, path, key, forceUpdate, shouldRefreshResources} = action.params
+  const {city, language, path, key, criterion} = action.params
   try {
-    yield call(loadCityContent, dataContainer, city, language, forceUpdate, shouldRefreshResources)
+    const loadCriterion = new ContentLoadCriterion(criterion)
+    const languageLoaded = yield call(loadCityContent,
```

Comment:
In fact this is rather a `cityContentLoaded` and not a `languageLoaded`

Reply:
Right now loadCityContent only returns false if the `language` is not available in the city. Therefore the name `languageLoaded` is oke.

Reply:
Oke I changed it :P

Reply:
:grimacing: